AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/HierarchicalVersionUtils.dfy

```diff
@@ -15,6 +16,8 @@ module HierarchicalVersionUtils {
   // BKC => Branch Key Context
   const BKC_DIGEST_LENGTH: uint8 := 48
   type PlainTextTuple = s: seq<uint8> | |s| == 80 witness *
+  import opened Seq
+  import CanonicalEncryptionContext
```

Comment:
Style/Blocking: We generally put imports before constants.

Reply:
those got in after auto merge